haas/client/switch.py

```diff
@@ -0,0 +1,58 @@
+import json
+from haas.client.base import ClientBase
+from haas.client import errors
+
+
+class Switch(ClientBase):
+    """ Consists of calls to query and manipulate node related
+    objects and relations """
+
+    def list(self):
+        """ List all nodes that HIL manages """
+        url = self.object_url('/switches')
+        return self.check_response(self.s.get(url))
+
+    def register(self, switch, subtype, *args):
+        """ Registers a switch with name <switch> and
+        model <subtype> , and relevant arguments  in <*args>
+        """
+#       It is assumed that the HIL administrator is aware of
+#       of the switches HIL will control and has read the
+#       HIL documentation to use appropriate flags to register
+#       it with HIL.
+
+        pass
```

Comment:
You don't need pass if you've got a docstring; please go through and remove it in cases like these where it isn't necessary.